locale/action_names.rb

```diff
@@ -0,0 +1,54 @@
+# Autogenerated!
+_("Update package")
```

Comment:
These look like they're from a plugin you have loaded using foreman-tasks, not foreman-tasks itself.  Wrong project?